fdbclient/ReadYourWrites.h

```diff
@@ -27,6 +27,9 @@
 #include "fdbclient/RYWIterator.h"
 #include <list>
 
+// Estimated number of overhead bytes per mutation.
+#define RYW_TRANSACTION_SIZE_OVERHEAD  30
```

Comment:
Where is this coming from?

Reply:
I inserted some mutations and obtain the size at commit time. This overhead is a rough approximation that is not very accurate.

Reply:
Looking into this a bit more, I think the expected size of an item in the transaction is the size of the item itself (e.g. `key.expectedSize()`) plus the size of the type being stored (for a `KeyValueRef`, `sizeof(KeyValueRef)`. This is based on the code from [here](https://github.com/apple/foundationdb/blob/7f427b60e2b8ad34af5bdf7b4059d4c84a7b9cf6/flow/Arena.h#L919). It seems like it would probably be relatively straightforward to add that in, so I'd be in favor of doing so rather than having a constant estimate.

Reply:
Sounds good. I've added `sizeof()` for better estimation.